li-apache-kafka-clients/src/main/java/com/linkedin/kafka/clients/consumer/LiKafkaConsumerConfig.java

```diff
@@ -92,6 +93,10 @@
 
   private static final String MAX_POLL_RECORDS_DOC = "The maximum number of records returned in a single call to poll().";
 
+  public static final String TOPIC_CREATION_POLL_INTERVAL_MS_DOC = "The frequency in milliseconds that a federated " +
```

Comment:
name should be more general, because this will be (maybe) used to update metadata in general for stuff like wildcard subscriptions as well )which could also be affected by topic deletion)

Reply:
Will use metadata.max.age.ms.